packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -327,11 +328,32 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       ...config,
       logger,
       web3,
+      getTime,
       vaultAbi: getAbi("VaultInterface", "latest"),
       erc20Abi: getAbi("IERC20Standard", "latest"),
       vaultAddress: config.address,
       blockFinder: getSharedBlockFinder(web3)
     });
+  } else if (config.type === "lp") {
+    const requiredFields = ["poolAddress", "tokenAddress"];
+    if (isMissingField(config, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating LPPriceFeed",
+      config
+    });
+
+    return new LPPriceFeed({
+      ...config,
+      logger,
+      web3,
+      getTime,
+      erc20Abi: getAbi("IERC20Standard", "latest"),
```

Comment:
Why don't you grab this same way as UniswapPriceFeed?

Reply:
Not super opinionated. Just figured this call was more directly what I needed since I don't need to construct a truffle contract to just get an ABI. Happy to change if you think `getTruffleContract` would be better.

Reply:
My guess is that the truffle one is that way because we did a large find-and-replace to change artifacts.require to getTruffleContract. However, now that we're using the core methods directly, getAbi is all that's needed here.

Reply:
OH! The truffle contract is already created at the top, so I can just pull it from there.

Reply:
Will update.

Reply:
Done.